profiles/source/dictionary/common.yaml

```
@@ -72,7 +72,7 @@ example:
       }
 homepage:
   title: Home Page
-  description: The home on the web that is related to this data package.
+  description: The home on the web that is related to this resource.
```

In `keywords` the common term `descriptor` is used. That is probably preferred over `resource`, since resource is often used for Data Resource.
```suggestion
  description: The home on the web that is related to this descriptor.
```